### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by Chakravarthi0**
Added card component.

### CODE REVIEW

**Comment by preethasri** [comment]
looks good to me 👍

**Comment by Swapnilkota108** [comment]
Good to go !!!

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+.card-horizontal-container {
+  display: flex;
+}
+
+.card-img-horizontal {
+  max-width: 30%;
+}
+
+.card-bordered {
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%), 0 6px 20px 0 rgb(0 0 0 / 19%);
```
Comment (mimansha-swarup) [inline components/card/card.css]: you can make variable for box shadow too

File: components/card/card.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        <p class="card-description">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Modi in
+          consequuntur quis.
+        </p>
+      </div>
+
+    </div>
+  </body>
+  <style>
+      #content{
```
Comment (mimansha-swarup) [inline components/card/card.html:185]: why this?

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: components/card/card.html [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+          </div>
+        </div>
+        <p class="card-description">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Modi in
+          consequuntur quis.
+        </p>
+        <div class="card-actions">
+          <button class="btn btn-primary">Read</button>
+          <button class="btn btn-secondary">Button</button>
+          <i class="card-icon material-icons">favorite_border</i>
```
Comment (srishti-maurya) [inline components/card/card.html:65]: Suggestion! You can align these icons a bit towards right, it would look great.

**Review by srishti-maurya — APPROVED** [review]
Components look good

File: components/card/card.html (hunk 2 above)
Comment (Chakravarthi0) [reply components/card/card.html:185]: I added those styles to make that particular page look better. please ignore that, I will do a better job with documentation site.

**Pull request merged**